app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -88,7 +90,42 @@ class BitcoinSServerMain(override val serverArgParser: ServerArgParser)(implicit
     }
 
     val peerSockets = {
-      nodeConf.peers.map(
+      val allPeers = {
+        if (nodeConf.network == MainNet) {
+          val dnsSeeds = Vector(
+            "seed.bitcoin.sipa.be",
```

Comment:
You should be able to use the seeds encoded in `NetworkParameters`. As a side note, that file hasn't been updated in a long time, so some of the seeds may need to be updated/replaced in `NetworkParameters.dnsSeeds` 

https://github.com/bitcoin-s/bitcoin-s/blob/8f958bf5d9957438d703819a087d87813f3edd89/core/src/main/scala/org/bitcoins/core/config/NetworkParameters.scala#L34